src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,39 @@
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__container {
+    max-width: 1200px;
+    padding: 0 50px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+  }
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+
+    img {
```

Comment:
It is better to use styling through classes and less to refer to tags directly.

Reply:
Still not fixed, don't style with tags, use classes instead.